2020-09-30 18:01:01.097 +03:00 [WRN] Failed to process the job '10394': an exception occurred. Retry attempt 2 of 10 will be performed in 00:01:14.
System.Net.WebException: The SSL connection could not be established, see inner exception. The remote certificate is invalid according to the validation procedure.
 ---> System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid according to the validation procedure.
   at System.Net.Security.SslStream.StartSendAuthResetSignal(ProtocolToken message, AsyncProtocolRequest asyncRequest, ExceptionDispatchInfo exception)
   at System.Net.Security.SslStream.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartReceiveBlob(Byte[] buffer, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartReceiveBlob(Byte[] buffer, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.CheckCompletionBeforeNextReceive(ProtocolToken message, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.StartSendBlob(Byte[] incoming, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.ProcessReceivedBlob(Byte[] buffer, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security.SslStream.PartialFrameCallback(AsyncProtocolRequest asyncRequest)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Security.SslStream.EndProcessAuthentication(IAsyncResult result)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean allowHttp2, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.FinishSendAsyncUnbuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at System.Net.HttpWebRequest.SendRequest()
   at System.Net.HttpWebRequest.GetResponse()
   --- End of inner exception stack trace ---
   at NewsUploader.NewsService.LoadNewsInDb(String url) in C:\Users\Lenovo\Documents\GitHub\SAU-TestRepository\GoodMoodProvider\NewsUploader\NewsService.cs:line 59
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
2020-09-30 18:01:07.618 +03:00 [INF] 30.09.2020 18:01:07|Info|Couldnt get body of https://rebenok.by/articles/stature/difficulty/28278-rebenok-plokho-uchitsya.html?utm_campaign=news-feed&utm_medium=rss&utm_source=rss-news
2020-09-30 18:01:22.760 +03:00 [INF] HTTP GET /api/News/GetAll responded 200 in 39802.0350 ms
2020-09-30 18:01:35.267 +03:00 [INF] HTTP GET /api/News/GetAll responded 200 in 38955.5765 ms
2020-09-30 18:01:37.432 +03:00 [INF] HTTP GET /api/News/GetAll responded 200 in 21132.1624 ms
2020-09-30 18:01:38.749 +03:00 [INF] 30.09.2020 18:01:38|Info|Couldnt get body of https://rebenok.by/articles/stature/healts/28279-koronavirus-i-deti.html?utm_campaign=news-feed&utm_medium=rss&utm_source=rss-news
2020-09-30 18:01:41.812 +03:00 [INF] 30.09.2020 18:01:41|Info|Couldnt get body of https://rebenok.by/inf/news_partners/28262-chto-nuzhno-vzyat-v-roddom-i-kak-nastroitsya-pered-otezdom-sprosili-u-mnogodetnoi-mamy.html?utm_campaign=news-feed&utm_medium=rss&utm_source=rss-news
2020-09-30 18:01:45.792 +03:00 [INF] 30.09.2020 18:01:45|Info|Couldnt get body of https://rebenok.by/inf/features/28284-my-protiv-nasiliya-i-zhestokosti-videoobrashchenie-mnogodetnykh-mam-belarusi.html?utm_campaign=news-feed&utm_medium=rss&utm_source=rss-news
